x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/action/EsqlQueryRequestTests.java

```diff
@@ -238,7 +238,6 @@ public void testSnapshotVersionIsOnlyValidOnSnapshot() throws IOException {
         assertThat(request.validate().getMessage(), containsString(errorOnNonSnapshotBuilds));
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/104890")
```

Comment:
Needs reverting as well before this gets merged.

Reply:
We're going to keep this one and just enable the validation.